### DISCUSSION TITLE
Use phoebe and the ellc backend

### DISCUSSION THREAD

**Discussion by HeBe-lab** [phoebe-project/phoebe2#530] [Questions]
Hi,I was wondering why the results from 'phoebe01'  are different from those from 'fastcompute' ? 
Phoebe01 yields significantly more accurate results.

**Comment by kecnry** [comment]
The phoebe 2 backend (which is labeled `phoebe01`, by default) and the ellc backend (which is a wrapper around an external code and labeled `fastcompute` in some of the tutorials) use different parameterizations and assumptions.  Depending on your system parameters, these can result in differing models, so its always a good idea to check them against each other before choosing ellc (or any of the other alternative backends) as a "faster" option.  

[This table](http://phoebe-project.org/docs/latest/examples/compute_comparison_table) provides a top-level overview of some of these differences.  You might also find [this tutorial from the past workshop](http://phoebe-project.org/workshops/2021june/Tutorial_optimizing_computations) helpful as it discusses some of the switches for assumptions within phoebe and how important it is to occasionally test any model with other backends or effects disabled against a robust (but slow) model.

Hope this helps!
Kyle

**Reply by HeBe-lab** [reply]
Thank you very much for your reply.
